### ISSUE TITLE
Support for structured append

### ISSUE THREAD

**Issue by shobhitvaish** [codebude/QRCoder#255]
## Type of issue

Request for new feature/improvement  

## Expected Behavior

Thanks for the great library. It is easy to use.

Do you have any plan to support structured append?

**Comment by codebude** [comment]
Currently structured appends are not on my road map, because they would bring in major changes in the library. But feel free to kickstart an implementation and send in a MR/PR. If your solution matches the style/conventions of this project, I'm willingly to merge your future developments.

**Comment by scherven** [comment]
@Shane32 are you still looking for help on this issue? I'm interested in helping with an implementation

**Comment by Shane32** [comment]
Yes please @scherven .  Thank you in advance for your assistance!
